### PR TITLE
[web] Directly import createElement

### PR DESCRIPTION
**Pull request by EvanBacon**
- Like this https://github.com/expo/expo/pull/8671 but for every instance of `createElement`.
- This may break snack since it can't handle things like fbjs

### CODE REVIEW

**Review by bbarthec — COMMENTED** [review]
LGTM, but what about this possibility to break snack?
Do we care here about it or you'll fix it later? 🤔 
Is it possible to test it locally? 🤔

File: packages/expo-camera/CHANGELOG.md [hunk 1]
```diff
@@ -7,12 +7,13 @@
...
 
 - Delete `prop-types` in favor of TypeScript. ([#8680](https://github.com/expo/expo/pull/8680) by [@EvanBacon](https://github.com/EvanBacon))
+- [camera] Directly import `createElement` from `react-native-web` for RNW v12 support. ([#8773](https://github.com/expo/expo/pull/8773) by [@EvanBacon](https://github.com/EvanBacon))
 
 ### 🐛 Bug fixes
 
 ## 8.3.1 — 2020-05-29
 
-*This version does not introduce any user-facing changes.*
+_This version does not introduce any user-facing changes._
```
Comment (lukmccall) [inline packages/expo-camera/CHANGELOG.md:16]: Same ^^^

File: packages/expo-av/CHANGELOG.md [hunk 2]
```diff
@@ -7,12 +7,13 @@
...
 
 - [av] Delete `prop-types` in favor of TypeScript. ([#8679](https://github.com/expo/expo/pull/8679) by [@EvanBacon](https://github.com/EvanBacon))
+- [av] Directly import `createElement` from `react-native-web` for RNW v12 support. ([#8773](https://github.com/expo/expo/pull/8773) by [@EvanBacon](https://github.com/EvanBacon))
 
 ### 🐛 Bug fixes
 
 ## 8.2.1 — 2020-05-29
 
-*This version does not introduce any user-facing changes.*
+_This version does not introduce any user-facing changes._
```
Comment (lukmccall) [inline packages/expo-av/CHANGELOG.md:16]: I think we don't want this change.

File: packages/expo-camera/CHANGELOG.md (hunk 1 above)
Comment (tsapeta) [reply packages/expo-camera/CHANGELOG.md:16]: We do, see here https://github.com/expo/expo/pull/8774

File: packages/expo-av/CHANGELOG.md [hunk 3]
```diff
@@ -7,12 +7,13 @@
 ### 🎉 New features
 
 - [av] Delete `prop-types` in favor of TypeScript. ([#8679](https://github.com/expo/expo/pull/8679) by [@EvanBacon](https://github.com/EvanBacon))
+- [av] Directly import `createElement` from `react-native-web` for RNW v12 support. ([#8773](https://github.com/expo/expo/pull/8773) by [@EvanBacon](https://github.com/EvanBacon))
```
Comment (tsapeta) [inline packages/expo-av/CHANGELOG.md:10]: I think we don't need to prefix entries by the package name like `[av]` 🤔 That's kinda obvious they refer to that package. Also, the script that merges changelog into SDK changelog groups them by package like here: https://github.com/expo/expo/blob/master/CHANGELOG.md#-bug-fixes-1

```suggestion
- Directly import `createElement` from `react-native-web` for RNW v12 support. ([#8773](https://github.com/expo/expo/pull/8773) by [@EvanBacon](https://github.com/EvanBacon))
```

**Review by tsapeta — APPROVED** [review]

File: packages/expo-camera/CHANGELOG.md (hunk 1 above)
Comment (lukmccall) [reply packages/expo-camera/CHANGELOG.md:16]: Oh, I see. Thanks ;)

**Review by lukmccall — APPROVED** [review]

File: packages/expo-av/CHANGELOG.md (hunk 3 above)
Comment (EvanBacon) [reply packages/expo-av/CHANGELOG.md:10]: the changelog danger script added the prefix

**Pull request merged**